src/glb-redirect/ipt_GLBREDIRECT.c

```diff
@@ -106,6 +106,13 @@ static unsigned int glbredirect_send_forwarded_skb(struct net *net, struct sk_bu
 		return NF_STOLEN;
 	}
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5,4,0)
+	/* Clear out timestamp to make forwarding work with fq on Linux 5.4:
+	 * - https://github.com/torvalds/linux/commit/9669fffc1415bb0c
```

Comment:
:+1: Agreed, the second answered my question "what do other forwarding implementations do?".
```suggestion
	 * - https://github.com/torvalds/linux/commit/8203e2d844d34af2
```